data/avro-schema/src/test/kotlin/net/corda/data/p2p/HostedIdentityEntryCompatibilityTest.kt

```diff
@@ -0,0 +1,64 @@
+package net.corda.data.p2p
+
+import net.corda.data.identity.HoldingIdentity
+import org.apache.avro.Schema
+import org.apache.avro.SchemaCompatibility
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+
+class HostedIdentityEntryCompatibilityTest {
+    @Test
+    fun `check HostedIdentityEntry schema changes between Corda 5_2 and 5_2_1 are compatible`() {
```

Comment:
Do we need to modify this test to check 5.3 instead?

Reply:
Doesn't look like it based on similar tests, for example - [flow status](https://github.com/corda/corda-api/blob/527bb42440bab8dda5efbd9f65d18f8ab775c6ef/data/avro-schema/src/test/kotlin/net/corda/data/flow/state/checkpoint/FlowStatusSchemaCompatibilityTest.kt#L15), [decline registration](https://github.com/corda/corda-api/blob/527bb42440bab8dda5efbd9f65d18f8ab775c6ef/data/avro-schema/src/test/kotlin/net/corda/data/membership/command/registration/mgm/DeclineRegistrationCompatibilityTest.kt#L11).